python/setup.py

```diff
@@ -122,6 +122,17 @@ def remove_if_exists(file_path):
         include_package_data=True,
         package_dir=PACKAGE_DIR,
         package_data=PACKAGE_DATA,
+        options={
+            "generate_py_protobufs": {
+                "source_dir": this_directory,
+                "output_dir": this_directory,
+                "proto_files": [
```

Comment:
I agree with it that we can avoid specifying `proto_files`, but I prefer to specify source_dir as `os.join(this_directory, 'feathub/common/protobuf')`, instead of `feathub/common/protobuf`. Because it can guarantee the setup.py can work correctly no matter under which directory it is executed, for example
```
python python/setup.py generate_py_protobufs
```

```
cd python
python setup.py generate_py_protobufs
```